django/test/testcases.py

```diff
@@ -54,9 +57,7 @@ def to_list(value):
     Put value into a list if it's not already one. Return an empty list if
     value is None.
     """
-    if value is None:
-        value = []
-    elif not isinstance(value, list):
```

Comment:
I'm not sure about these changes, they seem unnecessary. We can raise a deprecation warning when `None` is passed in `errors` even without them.